### DISCUSSION TITLE
Redundant mode for video calls

### DISCUSSION THREAD

**Discussion by Chris5342** [Ysurac/openmptcprouter#2588] [Show and tell]
I was having problems with glitches on video calls (and it seems others do to).
It looked like Starlink was occasionally glitching (maybe as it switched satellites).

I am now using redundant mode which is a nice OMR feature that is not much documented.

Redundant mode uses all your interfaces in parallel ie packets are sent down all links so if a pocket is dropped on one interface it does not matter as it will get there on another interface.

This makes it much more reliable at the expense of reducing the bandwidth and potentially adding cost if some interfaces are metered.

I turn on redundant mode before an important call and turn it off after.

To turn it on simply:

Network -> MPTCP
and
![Screenshot 2022-10-01 at 08 04 30](https://user-images.githubusercontent.com/105393840/193397443-05604d8a-0f4b-447a-9ac8-1771a9848dc1.jpg)
(REMEMBER TO TAKE A NOTE OF THE PREVIOUS SETTiNG!)

To turn it off set it back (BLEST for me)

Here is a graph of redundant mode in action:

![Screenshot 2022-07-27 at 21 49 58](https://user-images.githubusercontent.com/105393840/193397538-f7cb81b4-3f53-4486-b276-6a8d28c70a7a.jpg)

Note all three interfaces carrying about the same download traffic (ADSL struggling a bit)
Starlink and 4G are carrying the same upload traffic but ADSl nothing (which makes sense).

**Comment by winkmichael** [comment]
Thanks for sharing, I will be experimenting with this moving forward.

**Comment by marccolemont** [comment]
Thanks I'll try that with SRT video connections

**Comment by bobradamus** [comment]
Is this still available? I can't find the setting at all as shown above.

**Reply by Ysurac** [reply]
Redundant mode will be back in next release based on kernel 6.6.

**Reply by KevinashoUWU** [reply]
Hi! I just installed latest OMR v0.61 that uses Kernel 6.6, I see in the Scheduler list "mptcp_bpf_red.o" I select it and save n apply but it doesn't work, it keeps doing bonding as I can see in the bandwidth graph

**Reply by pauleharris** [reply]
Hi - same issue here.  Any update on the redundant mode?

**Reply by alpham0d3x** [reply]
> Redundant mode will be back in next release based on kernel 6.6.

Hi, any updates on the redundant mode? Especially for video calls. My current OMR configuration takes 10-15sec to reestablish the video call after master connection fails. Any tips on that?
